.storybook/main.js

```diff
@@ -2,32 +2,8 @@ const path = require('path');
 
 module.exports = {
   stories: ['../src/**/*.stories.@(js|mdx)', '../guidelines/**/*.stories.@(js|mdx)'],
-  addons: ['@storybook/addon-essentials', '@storybook/addon-a11y'],
-  webpackFinal: async (config, { configType }) => {
-    // Rey wrote this, blame him.
-    // Essentially, we want to keep all the rest of the
-    // rules present, add our own css rule, and ignore the existing css rule
-    // that's in here
-    newConfig = config.module.rules.filter((rule) => {
-      return String(rule.test) !== '/\\.css$/';
-    });
-    config.module.rules = newConfig;
-    config.module.rules.push({
-      test: /\.css$/,
-      include: path.resolve(__dirname, '../'),
-      use: [
-        'style-loader',
-        {
-          loader: 'postcss-loader',
-          options: {
-            config: {
-              path: './',
-            },
-          },
-        },
-      ],
-    });
-
+  addons: ['@storybook/addon-postcss', '@storybook/addon-essentials', '@storybook/addon-a11y'],
+  webpackFinal: async (config) => {
```

Comment:
this is all it needs to be replaced with (storybook post css addon), so definitely no complaints from me